src/apps/chifra/pkg/utils/utils.go

```diff
@@ -57,9 +57,9 @@ func TestLogArgs(name string, args []string) {
 		return
 	}
 
-	fmt.Fprintf(os.Stderr, "TIME ~ CLOCK - <INFO>  : %s\n", name)
+	fmt.Fprintf(os.Stderr, "INFO[DATE|TIME] %s\n", name)
```

Comment:
I think we should use logger.Log to be sure that we have consistent message format across the whole codebase

Reply:
Nice catch.

Reply:
I agree.